npm-scripts.mjs

```diff
@@ -441,7 +459,7 @@ function flatcWorker() {
 function testNode() {
 	logInfo('testNode()');
 
-	executeCmd(`jest --silent false --detectOpenHandles ${args}`);
+	executeCmd(`jest --silent false --detectOpenHandles --forceExit ${args}`);
```

Comment:
Please don't

Reply:
Garbage collection on addons does not happen at our will. The native addon will be closed once the JS instance is garbage collected.

Within the tests, the addon is not garbage collected so the tests won't finish unless this option is set.

Reply:
Anyway I'm rechecking this because I need to see how it is garbage collected.

Reply:
While it makes sense honestly I don't think that Node triggers garbage collection IMMEDIATELY as soon as an object is unreferenced. It may take some time depending on Node internal things and, if so, we will get random warnings here.

Reply:
Correct, it does not trigger GC at all. Setting a variable to  `undefined` just let the GC identify that the object is not used.

In this exact case, now the tests do finish while before they didn't unless the `--forceExit` option was set. This wouldn't be needed for real life, but we don't want the test to hang (CI would fail) nor use `--forceExit` since it could shadow real problems.